### PR TITLE
Show Shimmer background color on Mac

### PR DESCRIPTION
**Pull request by amgleitman**
### Platforms Impacted
- [ ] iOS
- [x] macOS
- [ ] win32 (Office)
- [ ] windows
- [ ] android

### Description of changes

This change ensures that a Shimmer component's background color gets displayed on macOS. We also make sure that nothing gets drawn outside the border given a large enough corner radius.

### Verification

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot 2023-12-14 at 12 29 17 PM | Screenshot 2023-12-14 at 12 29 27 PM |

Corner radius fix (pay attention to bottom-right corner)
| Before corner radius fix | After corner radius fix |
|-|-|
| Screenshot 2023-12-08 at 5 40 51 PM | Screenshot 2023-12-08 at 5 21 21 PM |

### Pull request checklist

This PR has considered (when applicable):
- [ ] Automated Tests
- [ ] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

File: packages/experimental/Shimmer/src/Shimmer.tsx [hunk 1]
```diff
@@ -120,12 +123,21 @@ export const Shimmer = compose<ShimmerType>({
...
           </Defs>
           <G transformProps={rtlTransfrom}>
+            <Rect
+              x="0"
+              y="0"
+              width={memoizedShimmerData.containerWidth}
+              height={memoizedShimmerData.containerHeight}
+              fill={memoizedShimmerData.backgroundColor}
+              rx={memoizedShimmerData.containerBorderRadius}
+            />
```
Comment (Saadnajmi) [inline packages/experimental/Shimmer/src/Shimmer.tsx:133]: Why the extra Rect?

File: apps/fluent-tester/src/TestComponents/Shimmer/ShimmerTest.tsx [hunk 2]
```diff
@@ -47,7 +47,7 @@ export const RoundedCornerClipCheckShimmer: React.FunctionComponent = () => {
         elements={shimmerRectsAndCircle()}
         duration={2000}
         delay={1000}
-        style={{ height: 100, width: 150, borderRadius: 16, borderWidth: 2, borderColor: 'black' }}
+        style={{ height: 100, width: 150, borderRadius: 20, borderWidth: 2, borderColor: 'black' }}
```
Comment (Saadnajmi) [inline apps/fluent-tester/src/TestComponents/Shimmer/ShimmerTest.tsx:50]: While we're at it, could we move all the inline styles to a stylesheet?

File: packages/experimental/Shimmer/src/Shimmer.tsx (hunk 1 above)
Comment (amgleitman) [reply packages/experimental/Shimmer/src/Shimmer.tsx:133]: This is what draws the background.

**Review by Saadnajmi — APPROVED** [review]

File: packages/experimental/Shimmer/src/Shimmer.tsx (hunk 1 above)
Comment (Saadnajmi) [reply packages/experimental/Shimmer/src/Shimmer.tsx:133]: ahh, thanks!

**Pull request merged**